app/os_android.go

```diff
@@ -552,7 +552,7 @@ func Java_org_gioui_GioView_onLowMemory(env *C.JNIEnv, class C.jclass) {
 func Java_org_gioui_GioView_onConfigurationChanged(env *C.JNIEnv, class C.jclass, view C.jlong) {
 	w := cgo.Handle(view).Value().(*window)
 	w.loadConfig(env, class)
-	if w.stage >= system.StageRunning {
+	if w.stage > system.StagePaused {
```

Comment:
Nit: it seems a bit more clear to replace with `w.stage >= system.StageInactive`. Here and elsewhere.